src/components/shared/MarkerTooltipContents.js

```diff
@@ -122,6 +122,29 @@ function _makePhaseTimesArray(
   return array;
 }
 
+function _dataPriorityReplace(priority: number): ?string {
+  var prioLabel;
+
+  // https://searchfox.org/mozilla-central/source/xpcom/threads/nsISupportsPriority.idl#24-28
+  if(priority < -10) {
+    prioLabel = 'Highest';
+  } else if ((priority > -11) && (priority < 0)){
+    prioLabel = 'High';
+  } else if (priority === 0) {
+    prioLabel = 'Normal';
+  } else if ((priority < 11) && (priority > 0)){
```

Comment:
for the same reason, please put the left bound first (so the comparison "0" first), so that we can read it like "if (0 <= priority < 11)"